tests/run_tests.py

```
@@ -19,66 +19,71 @@
     # Check if Ollama is running
     try:
         import requests
-        response = requests.get('http://localhost:11434/api/version', timeout=2)
+
+        response = requests.get("http://localhost:11434/api/version", timeout=2)
         ollama_available = response.status_code == 200
-        print(f"{'✓' if ollama_available else '⚠'} Ollama service: {'Running' if ollama_available else 'Not running'}")
+        print(
+            f"{'✓' if ollama_available else '⚠'} Ollama service: {'Running' if ollama_available else 'Not running'}"
+        )
     except:
```

Ruff (0.14.6)

Do not use bare `except`

(E722)